types/pouchdb-core/index.d.ts

```diff
@@ -55,6 +55,15 @@ declare namespace PouchDB {
             rev: RevisionId;
             status: Availability;
         }
+        interface RevisionDiffList {
+            [documentId : string]: string[];
+        }
+        interface RevisionDiff {
+            [change : string]: string[];
```

Comment:
Nop, just tried it, the compiler doesn't even try to traverse that type if it differs from `number | string`.